lib/whitehall/document_collection_group_membership_updater.rb

```diff
@@ -0,0 +1,28 @@
+module Whitehall
+  class DocumentCollectionGroupMembershipUpdater
```

Comment:
I don't know if I'm opening a can of worms here but I wonder if the class name here captures what we're doing properly. I wonder if we want to cover that this is for a migration and/or one to content publisher. 

Maybe we should have some of these classes under a ContentPublisherMigration namespace? What do you think?